frontend/src/features/properties/components/forms/subforms/PagedBuildingForms.tsx

```diff
@@ -33,6 +34,18 @@ const getPageErrors = (errors: any, nameSpace: any) => {
   );
 };
 
+const deletePagedBuilding = (
+  buildings: IFormBuilding[],
+  currentPage: number,
+  setCurrentPage: Function,
+  arrayHelpers: any,
+) => {
+  if (currentPage === buildings.length - 1) {
```

Comment:
Does this work ok with a single building?

Reply:
yup, seems to be working. need to implement changes for the conditional dirty render we talked about below though